src/KeyVault/KeyVault/help/Get-AzKeyVaultCertificate.md

```diff
@@ -112,7 +112,7 @@ This command gets the certificate named `TestCert01` from the key vault named `C
 ```powershell
 $CertBase64 = Get-AzKeyVaultSecret -VaultName $vaultName -Name $certName -AsPlainText
 $CertBytes = [Convert]::FromBase64String($CertBase64)
-Set-Content -Path cert.pfx -Value $CertBytes -AsByteStream
+Set-Content -Path cert.pfx -Value $CertBytes -Encoding Byte
```

Comment:
Don't find Byte as an accepted value of `-Encoding`

Reply:
As it works fine with Powershell Core (even though most of people use Windows Powershell and not Powershell Core and Powershell Core(v7) is retro compatible with older version) so i'll resolve the conversation.

Thank you for your review and have a nice day. 👍